src/app/components/hmcts-global-header/hmcts-global-header.component.ts

```diff
@@ -3,8 +3,10 @@ import { FeatureToggleService } from '@hmcts/rpx-xui-common-lib';
 import { Store } from '@ngrx/store';
 import { BehaviorSubject, Observable, of } from 'rxjs';
 import { map, switchMap, tap } from 'rxjs/operators';
+import { SearchStatePersistenceKey } from 'src/search/enums';
 import * as fromAppStore from '../../../app/store';
 import * as fromNocStore from '../../../noc/store';
+import { SearchService } from '../../../search/services/search.service';
 import { NavItemsModel } from '../../models/nav-item.model';
```

Comment:
@johnbenjamin-hmcts - please group `NavItemsModel` and `UserNavModel` into a single import.

Reply:
Thank you @Daniel-Lam, I have applied the changes